### DISCUSSION TITLE
Push done by Github Action does not create a "pushEvent" in API response for Events.

### DISCUSSION THREAD

**Discussion by Prakash4844** [community/community#59063] [Apps, API and Webhooks] [answered]
### Select Topic Area

General

### Body

So, I have a python script that commits to repo and is scheduled to run via cron through Github Action, The problem is even though the commits happens and is counted on github and commits are available in repo. if i call the Github Events API, it doesn't contain the pushEvent for the commit made by github action.
Github action is using My Email to Commit instead of standard github's provided "no-reply Email.

Here's API response Screenshot 
![image](https://github.com/community/community/assets/81550376/3f8320cb-7745-405d-bbdb-c4cb4aee8b02)

And here's commit in repo and contribution graph
![image](https://github.com/community/community/assets/81550376/2b4b90a7-0b52-4206-89bc-5367ad85efad)

![image](https://github.com/community/community/assets/81550376/f1da5651-8e5b-4f62-94b3-17f15d3d07cf)

Why could this be happening? if i push manually the pushEvent is created Normally.

**Comment by Prakash4844** [comment, marked as answer]
Fixed it, see this. 

https://github.com/Prakash4844/Github_Streak_Maintainer/issues/18#issuecomment-1606061902

**Comment by django9847** [comment]
@Prakash4844 , I was working on the local branch and raised a PR. After raising the PR, i have added few commits to my same local branch. Although, I push my changes to same but couldn't see my newly added commits in the PR. However, my local commits history says that branch is up-to-date. Could you please help me in the same.
My github PR : https://github.com/CSCI-GA-2820-SU23-001/inventory/pull/19

**Reply by Prakash4844** [reply]
I think you have replied to wrong thread, i do not work on that repo and have no idea what it is doing.
However, Check if you PR is merged if yes and then you pushed changes to remote in that case you might need to do PR again, if commit doesn't show up on remote please check if you have pushed your branch to remote and and it is up to date with local branch.
